pkg/storage/engine/rocksdb.go

```diff
@@ -80,6 +80,30 @@ var rocksdbConcurrency = envutil.EnvOrDefaultInt(
 		return max
 	}())
 
+var ingestDelayL0Threshold = settings.RegisterIntSetting(
+	"rocksdb.ingest_backpressure.l0_file_count_threshold",
+	"number of L0 files after which to backpressure SST ingestions",
+	20,
+)
+
+var ingestDelayPerFile = settings.RegisterDurationSetting(
+	"rocksdb.ingest_backpressure.delay_l0_file",
+	"delay to add to ingestions per file in L0 over the configured limit",
+	time.Millisecond*200,
```

Comment:
Have you considered deriving these values from RocksDB's rates/limits? For example the Rocks slowdown rate is currently configured at 16MB/s, so if we had a 64MB file to ingest, we'd calculate a four second delay.